### ISSUE TITLE
Large

### ISSUE THREAD

**Issue by ssphillips** [michaeljmartin/dho-backup#2]
Hey there, I notice in your readme you have a Too item to "Auto-split large files and use multi-part upload". Did you ever implement this? I will definitely need it for some of my sites...

Thanks!

**Comment by michaeljmartin** [comment]
Howdy!

This repo was really just a pet project of mine that I built when I was just learning to program and use git. The code is therefore a bit mediocre/not likely fit for production use, and it's been a long while since I've had time to maintain or update it. In addition to the issue of large files needing to be split, there's no restore method in the code anywhere (not that you won't be able to get your data back, it's just not automated).

I have implemented multi-part upload in an internal script at work, and it was not at all difficult to get it right. The key was making use of the [StringIO](https://docs.python.org/2/library/stringio.html) library. If you want to take a look at a working implementation of multi-part upload, the [Dreamhost fork of boto-rsync](https://github.com/dreamhost/boto_rsync) uses it.

Michael
